app/src/androidTest/java/tech/relaycorp/gateway/background/endpoint/NounceRequestServiceTest.kt

```diff
@@ -0,0 +1,48 @@
+package tech.relaycorp.gateway.background.endpoint
+
+import android.content.Context
+import android.content.Intent
+import android.os.Handler
+import android.os.Looper
+import android.os.Message
+import android.os.Messenger
+import androidx.test.core.app.ApplicationProvider.getApplicationContext
+import androidx.test.rule.ServiceTestRule
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Rule
+import org.junit.Test
+
+class NounceRequestServiceTest {
```

Comment:
Can you please add a test to make that nothing breaks, hangs or times out when sending an invalid request? (I.e., one without `NounceRequestService.NOUNCE_REQUEST`)

Reply:
Ok.